.gitignore

```diff
@@ -9,3 +9,4 @@ terraform.tfstate
 terraform.tfstate.backup
 terraform.tfvars
 .terraform*
+aws-ecsfargate-terraform/op-scim.plan
```

Comment:
Very minor nitpick: I'd shorten this to `*.plan`.

Reply:
Done.